lib/date_range.rb

```diff
@@ -0,0 +1,24 @@
+module HotelManagementSystem
+    class DateRange
+        def self.is_valid?(start_date, end_date)
```

Comment:
I'd like to see this turned into a constructor that `raise`s since a date range is a thing you can perform meaningful operations on.